src/api/controllers/db-controller.ts

```diff
@@ -992,6 +992,10 @@ function parseContractCallMetadata(
   if (!contract.found) {
     throw new Error(`Failed to lookup smart contract by ID ${parsedTx.contract_call.contract_id}`);
   }
+  if (!contract.result.abi)
```

Comment:
Small style nit to avoid possible future confusion: can you keep the curly braces here?